.github/pull_request_template.md

```diff
@@ -1,61 +1,36 @@
-
-Before opening a PR, please note these guidelines:
-
-- Each PR should only address ONE topic and have an associated GitHub issue
-- No hardcoded or paths to personal directories should be present
-- No temporary or backup files should be committed
-- Any code that was disabled by being commented out should be removed
-- Please provide a on-line, but descriptive title to the PR e.g. Adds XYZ functionality or Fixes ABC bugs, etc.  DO NOT use branch names as titles.
--->
 
-**Description**
+INSTRUCTIONS
 
-<!-- Please include relevant motivation and context. -->
-<!-- Please include a summary of the change and which issue is fixed. -->
-<!-- List any dependencies that are required for this change. -->
+- Please do not commit temporary, backup, or binary files.
+- Please remove commented-out code.
+- Please ensure code, config files, etc., contain no hardcoded paths.
+- Please format code snippets in PR description/comments with ```code block``` or `inline code`.
+- Please consider adding your own review comments to guide other reviewers.
+
+-->
 
-<!-- Please provide reference to the issue this pull request is addressing. -->
-<!-- For e.g. Fixes #IssueNumber -->
+**Description**
 
-**Type of change**
+<!-- A summary of the change, including relevant motivation, context, useful links, etc. -->
 
-Please delete options that are not relevant.
+**Type**
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
+Select one or more:
 
-**How Has This Been Tested?**
+- [ ] Bug fix (corrects a known issue)
+- [ ] Code maintenance (refactoring, etc.)
+- [ ] New feature (adds functionality)
 
-<!-- Please describe the tests that you ran to verify your changes. -->
-<!-- Provide instructions so we can reproduce. -->
-<!-- Please also list any relevant details for your test configuration -->
+**Impact**
 
-<!-- Use the following as a guide to list your tests and delete options that are not relevant. Expand as necessary. -->
-<!--
-- [ ] pytests in GitHub actions.
-- [ ] Tests on XYZ OS/HPC/CSP
--->
+Select one:
 
-**Checklist**
+- [ ] This is a non-breaking change (existing functionality continues to work as expected)
+- [ ] This is a breaking change (changes existing functionality)
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] My changes need updates to the documentation.  I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] New and existing tests pass with my changes
-- [ ] Any dependent changes have been merged and published
+**Documentation**
 
-<!-- 
- If there is a co-author attribution:
```

Comment:
Apart from being polite, what does/did this accomplish? I couldn't find any observable effect. It would, of course, be nice to acknowledge help from colleagues, but I'm not sure that merits an explicit PR step. Does anyone know differently?

Reply:
I feel like this was mostly relevant when there was a constant collaboration on tickets. i.e. Emily and I working together on tickets during our time on the team as interns, same can be said for Janet and Emily working as interns on the team and for future intern collaboration. Also was useful for the first 2 months when Emily and I were new full-time developers with the team.

For current purposes, I agree it's nice to be polite and if discussion is needed on a closed ticket where the main author has the left the team but the co-author has not.

Reply:
So do you think the goal here was just to record in the PR who else worked on it? I got the feeling that maybe there was some kind of automation that was processing this, given the explicit instructions of exact format, one co-author per line, email address has to be the GitHub-registered email address, etc. Why all that specificity otherwise?

Reply:
Yes, I would think that the goal in the past was to record in the PR who else had worked on it. I believe the intended goal is/was to push your final commit to GitHub with the co-author as an option included on the command line. I think this was an idea implemented during the spring semester right before we were brought on to the team.

Reply:
Yep - like a lot of the other comments, they were at times more for user onboarding and defining the standards than providing information to reviewers. I think we're at a good place to transition to more external-facing options

Reply:
Janet and I would alternate who did the driving ticket by ticket, so when we had a PR, only one of our names would be on it, despite pairing for every part of it. This was our way of giving credit to the other person. I agree it's unnecessary now. If we have two interns come onto the team in the future, I would think the better approach is to simply show them how to work with remote forks.

Reply:
I do want to make sure that when we do have people pair, that they have demonstrable evidence that they were involved in contributions during their time on the project.

Reply:
That makes sense. The PR has an _Assignees_ attribute, where you can add people who worked on the PR. Would that be a workable mechanism?

Reply:
GitHub describes that field as
> Assignees clarify who is working on specific issues and pull requests.
Sounds about right to me.

Reply:
I'm going to add it. We can remove it if we don't like it.